src/Common/tests/Tests/Interop/Windows/mincore/GetFullPathName.Tests.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text;
+using Xunit;
+
+namespace Tests.Interop.Windows
+{
+    public class GetFullPathNameTests
+    {
+        [PlatformSpecific(PlatformID.Windows)]
+        [Theory]
+        [InlineData(@"C:\", @"C:\")]
+        [InlineData(@"C:\.", @"C:\")]
+        [InlineData(@"C:\..", @"C:\")]
+        [InlineData(@"C:\..\..", @"C:\")]
+        [InlineData(@"C:\A\..", @"C:\")]
+        [InlineData(@"C:\..\..\A\..", @"C:\")]
+        public static void GetFullPathName_Windows_RelativeRoot(string path, string expected)
```

Comment:
That we have all of these Interop.Windows.mincore tests is I think another indicator that all of this logic should be in PathInternal rather than in Interop.mincore.